client/my-sites/site-settings/form-writing.jsx

```diff
@@ -237,13 +255,14 @@ export default connect(
 
 		return {
 			jetpackCustomTypesModuleActive: false !== isJetpackModuleActive( state, siteId, 'custom-content-types' ),
+			jetpackCarouselModuleActive: false !== isModuleActive( state, siteId, 'carousel' ),
```

Comment:
I agree, see my review comment about this: https://github.com/Automattic/wp-calypso/pull/9802#discussion_r90853483